front/app/containers/Admin/initiatives/settings/PageBody.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
```

Comment:
This component is used to edit the contents of the 'Proposals information page' (`/pages/initiatives`). This page can not be edited from other places, like the 'Navigation' tab.